README.md

```diff
@@ -1 +1,115 @@
-# leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Leaderboard <a name="about-project"></a>
+
+The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+## 🛠 Built With <a name="built-with"></a>
+
+<details>
+  <summary>Page was built using basic HTML and CSS, ES6 modules JavaScript</summary>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **Project has ES6 modules**
+- **Await and async has been used for API's**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you remove the above buttons since both work the same way so it would be great if you remove one of them to make your readme look professional 👍